src/main/java/com/zufar/onlinestore/payment/api/PaymentApi.java

```diff
@@ -15,35 +12,31 @@ public interface PaymentApi {
      *
      * @param createPaymentDto the request dto to create a payment object
      * @return PaymentDetailsWithTokenDto combines payment details and a payment token for payment processing on the front end side
-     * @throws StripeException this is the parent of all Stripe API exceptions, in this method, exceptions descendants related to the payment creation and processing
      * */
-    PaymentDetailsWithTokenDto createPayment(final CreatePaymentDto createPaymentDto) throws StripeException;
+    PaymentDetailsWithTokenDto createPayment(final CreatePaymentDto createPaymentDto);
```

Comment:
please, leave `throws StripeException`
Yes, you catch all these exceptions with `ExceptionsHandler`, but it is valid only for your endpoint. What if we would like to use this API in our backend? 
Me as a backend developer would like to know that your API can throw an exception. That is why it makes sense to provide this info in your API that this method `throws StripeException`

Reply:
done